src/main/java/net/openhft/chronicle/bytes/HexDumpBytes.java

```diff
@@ -273,12 +277,14 @@ public long capacity() {
     @Override
     public long addressForRead(long offset)
```

Comment:
Should we add @NonNegative annotation to these for consistency?

Reply:
Agree. I believe that offsets should always be from the start of the Bytes and thus always be >= 0

Reply:
I think we should. This makes the contract clearer. I have raised https://github.com/OpenHFT/Chronicle-Bytes/issues/290